charts/core-crd/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: core-crd
 description: A Helm chart for Kubernetes
 type: application
-version: 2.12.6
+version: 2.12.7
 appVersion: "v2.12.6"
```

Comment:
Need to update appVersion for all in internal registry @jasmingacic

Reply:
should it be 2.12.8?

Reply:
I don't know, can you check?